doc/api/events.md

```diff
@@ -703,11 +703,15 @@ added: v11.13.0
 * `name` {string}
 * Returns: {Promise}
 
-Creates a `Promise` that is resolved when the `EventEmitter` emits the given
+Creates a `Promise` that is fulfilled when the `EventEmitter` emits the given
 event or that is rejected when the `EventEmitter` emits `'error'`.
 The `Promise` will resolve with an array of all the arguments emitted to the
 given event.
 
+Note: This method is intentionally generic and works with the web platform
+[EventTarget](WHATWG-EventTarget) interface, which have no special
+`'error'` event semantics and do not listen to the `'error'` event.
```

Comment:
Unless I'm missing something, an example showing the use of the EventTarget API would be helpful.